packages/terra-form-select/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Unreleased
 ----------
 ### Fixed
 * Added CSS style to display the select box-shadow over the toggle border
+* Added CSS custom property to enable theming of disabled state arrow icon
```

Comment:
Both of these change log updates should probably be under an `Added` section. Not a `Fixed`

Reply:
done 6b3be35a